frontend/app/work_packages/controllers/dialogs/columns.js

```diff
@@ -55,10 +58,12 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
     return _.difference(vm.selectedColumns, vm.oldSelectedColumns);
   }
 
+  function getColumnName(column) {
```

Comment:
It saved me from writing the same inline function - which looks horrible in overly verbose Javascript - twice.
Probably saved me effectively a line while making the code more readable IMO.